client/src/Components/RegistrationForm/RegistrationForm.tsx

```diff
@@ -64,8 +57,10 @@ export const RegistrationForm = ({ onSubmit }: RegistrationFormType) => {
           type="name"
           onChange={formik.handleChange}
           data-testid="inputId"
-          value={formik.values.name} />
-        <RegistrationErrorMessage>{formik.errors.name && formik.touched.name ? formik.errors.name : null}
+          value={formik.values.name}
+        />
+        <RegistrationErrorMessage>
+          {formik.errors.name && formik.touched.name ? formik.errors.name : null}
```

Comment:
Wydeje mi się że samo formik.errors.name tutaj powinno działać dobrze, a jak nie to:
formik.touched.name ? formik.errors.name : null i to powinno być krótsze :D

Reply:
W sumie to mona to zupenie inaczej rozwizac wystarczy sam komponent Error z formika. @WarmuzWojtek co myslisz bo nie wiem jaka byla twoj pomysl na to dokladnie ?

Reply:
Znaczy mi chodziło tylko o skrócenie kodu do tego co jest bo wydaje mi się że ta krótsza wersja by działała identycznie